demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -30,14 +31,21 @@ def __init__(self, pack: str, update_type: None, pack_files: set, added_files: s
         self.pack_files = pack_files
         self.added_files = added_files
         self.pre_release = pre_release
+        self.specific_version = specific_version
+        self.existing_rn_changed = False
 
     def execute_update(self):
         if self.pack in IGNORED_PACK_NAMES:
             print_warning(f"Release notes are not required for the {self.pack} pack since this pack"
                           f" is not versioned.")
         else:
             try:
-                new_version, new_metadata = self.bump_version_number(self.pre_release)
+                if self.is_bump_required():
+                    self.update_type = "revision"
+                new_version, new_metadata = self.bump_version_number(self.specific_version, self.pre_release)
+                if self.is_bump_required():
+                    print_color(f"Changes were detected. Bumping {self.pack} to version: {new_version}",
+                                LOG_COLORS.NATIVE)
```

Comment:
this code is strange
why do you check is_bump_required, than update bump version than check again is_bump_required?